src/main/java/core/basesyntax/dao/CustomDaoClass.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.lib.Dao;
+import core.basesyntax.model.Bet;
+
+import java.util.List;
+
+@Dao
+public class CustomDaoClass implements BetDao {
```

Comment:
let's create your own interface and implement it here. For example: PersonDao and PersonDaoImpl

name  of this class isn't correct, we shouldn't use word `Class` in the name of the class